src/index.ts

```diff
@@ -1,11 +1,11 @@
-export { default, DataGridProps, DataGridHandle } from './DataGrid';
 export { default as Cell } from './Cell';
-export { default as Row } from './Row';
-export { default as HeaderCell } from './HeaderCell';
-export * from './Columns';
-export * from './Cell/cellContentRenderers';
 export { default as CellContent } from './Cell/CellContent';
-export * from './formatters';
+export * from './Cell/cellContentRenderers';
+export * from './Columns';
 export * from './common/editors';
 export * from './common/enums';
 export * from './common/types';
+export { DataGridHandle, DataGridProps, default } from './DataGrid';
+export * from './formatters';
+export { default as HeaderCell } from './HeaderCell';
+export { default as Row } from './Row';
```

Comment:
Why rearrange the exports? 😐

Reply:
same from the auto order cmd, it just sort the import export alphabetically.